packages/debugger/package.json

```diff
@@ -25,6 +25,8 @@
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-switch": "^1.0.3",
     "@radix-ui/react-tabs": "^1.0.4",
+    "@rainbow-me/rainbowkit": "^2.0.1",
```

Comment:
@stephancill how can I run this locally? There is an error when I run the dev mode `Error: No projectId found. Every dApp must now provide a WalletConnect Cloud projectId to enable WalletConnect v2 https://www.rainbowkit.com/docs/installation#configure`. 

Does it mean that I can't use debugger without this? If this is required only for transactions, could it be optional?